spring-boot-autoconfigure/src/test/java/org/axonframework/springboot/JdbcAutoConfigurationTest.java

```diff
@@ -89,6 +90,18 @@ void customTokenSchema() {
                 });
     }
 
+    @Test
```

Comment:
Would you mind adding another test validating the default `EventSchema` bean is used in absence of a custom `EventSchema bean?

Reply:
Test added